screenly-cast/inc/functions.php

```diff
@@ -0,0 +1,278 @@
+    $srly_h1_font_weight = !empty($srly_h1_font_weight_db)? $srly_h1_font_weight_db."px":"";
+	
+    $srly_time_font_size= !empty($srly_time_font_size_db)? $srly_time_font_size_db."px":"";
+	
+    $srly_content_margin_top= !empty($srly_content_margin_top_db)?  $srly_content_margin_top_db."px":"";
+    $srly_content_line_height= !empty($srly_content_line_height_db)? $srly_content_line_height_db."px":"";
+    $srly_content_font_size= !empty($srly_content_font_size_db)? $srly_content_font_size_db."px":"";
+	
+    $srly_blockquote_font_size= !empty( $srly_blockquote_font_size_db)? $srly_blockquote_font_size_db."px":"";
+	
+    $srly_h23456_margin = !empty( $srly_h23456_margin_db)? $srly_h23456_margin_db."px":"";
+	
+    $srly_h2_font_size= !empty($srly_h2_font_size_db)? $srly_h2_font_size_db."px":"";
+    $srly_h4_font_size= !empty($srly_h4_font_size_db)? $srly_h4_font_size_db."px":"";
+    $srly_h5_font_size= !empty( $srly_h5_font_size_db)? $srly_h5_font_size_db."px":"";
+    $srly_h6_font_size= !empty( $srly_h6_font_size_db)? $srly_h6_font_size_db."px":"";   
+    
+    $srly_body_font_family = $srly_body_font_family;
+    $srly_body_background=get_option('srly_body_background');
+	
+    $srly_brand_logo_display=get_option('srly_brand_logo_display');
+	
+    $srly_h1_padding = get_option('srly_h1_padding');
+    $srly_h1_color=get_option('srly_h1_color');
+	
+    $srly_time_color=get_option('srly_time_color');
+    $srly_time_display=get_option('srly_time_display');
+    $srly_time_font_weight=get_option('srly_time_font_weight');
+	
+    $srly_a_color=get_option('srly_a_color');
+    $srly_a_text_decoration=get_option('srly_a_text_decoration');
+    $srly_a_font_weight=get_option('srly_a_font_weight');
+    $srly_a_font_size=get_option('srly_a_font_size')."px";
+	
+    $srly_content_color=get_option('srly_content_color');
+    $srly_content_font_weight=get_option('srly_content_font_weight');
+	
+    $srly_blockquote_font_weight=get_option('srly_blockquote_font_weight');
+    $srly_blockquote_font_style=get_option('srly_blockquote_font_style');
+    $srly_blockquote_letter_spacing=get_option('srly_blockquote_letter_spacing');
+	
+    $srly_h23456_font_weight = get_option('srly_h23456_font_weight');
+	
+    $srly_h5_color = get_option('srly_h5_color');
+    $srly_h5_font_weight=get_option('srly_h5_font_weight');
+    $srly_h6_font_weight= get_option('srly_h6_font_weight');
+    $srly_h6_text_transform= get_option('srly_h6_text_transform');
+	
+    $srly_b_strong_font_weight= get_option('srly_b_strong_font_weight');
+	
+    $srly_ul_ol_padding_left= get_option('srly_ul_ol_padding_left');
+    $srly_ul_list_style_type= get_option('srly_ul_list_style_type');
+    $srly_ul_li_padding_left= get_option('srly_ul_li_padding_left');
+	
+    $srly_h3_color = get_option('srly_h3_color');
+    $srly_h3_font_size= get_option('srly_h3_font_size');
+    $srly_h3_font_weight =  get_option('srly_h3_font_weight');
+    
```

Comment:
It's nonoptimal mode to store setting in data base. It's better to use serialized array